model-optimizer/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -249,6 +249,25 @@ def serialize_meta_list(graph, node, schema, element, edges, unsupported):
         serialize_node_attributes(graph, item, [sub_schema], element, edges, unsupported)
 
 
+def serialize_runtime_info(node, parent_element: Element):
```

Comment:
nit: maybe `emmiter_test.py` should be updated with a couple of cases?

Reply:
Added serialization tests.